src/components/templates/index.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+import TableRow from "../molecules/TableRow";
+import Table from "../organisms/Table";
+
+const Index = () => {
+  // emulate a response from API of a json of projects
+  const [projects] = useState([
+    {
+      id: 1,
+      title: "My first project",
+      description: "testing propuse",
+      createdAt: Date.now(),
+    },
+    {
+      id: 2,
+      title: "My second project",
+      description: "testing propuse",
+      createdAt: Date.now(),
+    },
+    {
+      id: 3,
+      title: "My third project",
+      description: "testing propuse",
+      createdAt: Date.now(),
+    },
+    {
+      id: 4,
+      title: "My fourt project",
+      description: "testing propuse",
+      createdAt: Date.now(),
+    },
+  ]);
+
+  return (
+    <Table title="Projects">
+      {projects?.map((val) => (
+        <TableRow
+          title={val.title}
+          description={val.description}
```

Comment:
Now there is an error in console, you can add `key={val.id}`
<img width="685" alt="image" src="https://user-images.githubusercontent.com/79587920/226348684-025e5d09-725a-4678-86f4-210d830d1ff3.png">